kanvas/src/commonTest/kotlin/Call.kt

```diff
@@ -1,8 +1,6 @@
 package com.juul.krayon.kanvas
 
-import kotlin.reflect.KCallable
-
 data class Call(
-    val callable: KCallable<*>,
+    val callableName: String,
```

Comment:
Out of curiosity, what was it that prevented this from working? It's seems pretty convenient in terms of being able to refactor function names without breaking the tests.

Reply:
Internal compiler errors in the JS side of the IR compiler. There isn't much in the way of workarounds other than don't use function references, or don't update Kotlin.

Since this is test-only code, I didn't want to hold off on the Kotlin version bump altogether. But in the future, this is something that should be something we revert.